components/eam/src/physics/crm/cospsimulator_intr.F90

```diff
@@ -0,0 +1,3184 @@
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclcalipsotmpice (time,alt40,loc)
+       call addfld('CLD_CAL_TMPICE', (/'cosp_ht'/), 'A','percent', 'NOT SURE WHAT THIS IS Cloud Fraction',     &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclcalipsotmpun (time,alt40,loc)
+       call addfld('CLD_CAL_TMPUN', (/'cosp_ht'/), 'A','percent', 'NOT SURE WHAT THIS IS Cloud Fraction',      &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcltcalipsoice (time,loc)
+       call addfld('CLDTOT_CAL_ICE', horiz_only,'A','percent','Calipso Total Ice Cloud Fraction',    &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcltcalipsoliq (time,loc)
+       call addfld('CLDTOT_CAL_LIQ', horiz_only,'A','percent','Calipso Total Liquid Cloud Fraction', &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcltcalipsoun (time,loc)
+       call addfld('CLDTOT_CAL_UN',horiz_only,'A','percent','Calipso Total Undefined-Phase Cloud Fraction',      &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclhcalipsoice (time,loc)
+       call addfld('CLDHGH_CAL_ICE',horiz_only,'A','percent','Calipso High-level Ice Cloud Fraction',            &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclhcalipsoliq (time,loc)
+       call addfld('CLDHGH_CAL_LIQ',horiz_only,'A','percent','Calipso High-level Liquid Cloud Fraction',         &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclhcalipsoun (time,loc)
+       call addfld('CLDHGH_CAL_UN',horiz_only,'A','percent','Calipso High-level Undefined-Phase Cloud Fraction', &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclmcalipsoice (time,loc)
+       call addfld('CLDMED_CAL_ICE',horiz_only,'A','percent','Calipso Mid-level Ice Cloud Fraction',             &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclmcalipsoliq (time,loc)
+       call addfld('CLDMED_CAL_LIQ',horiz_only,'A','percent','Calipso Mid-level Liquid Cloud Fraction',          &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lclmcalipsoun (time,loc)
+       call addfld('CLDMED_CAL_UN',horiz_only,'A','percent','Calipso Mid-level Undefined-Phase Cloud Fraction',  &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcllcalipsoice (time,loc)
+       call addfld('CLDLOW_CAL_ICE',horiz_only,'A','percent','Calipso Low-level Ice Cloud Fraction',             &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcllcalipsoliq (time,loc)
+       call addfld('CLDLOW_CAL_LIQ',horiz_only,'A','percent','Calipso Low-level Liquid Cloud Fraction',          &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! lcllcalipsoun (time,loc) !+cosp1.4
+       call addfld('CLDLOW_CAL_UN',horiz_only,'A','percent','Calipso Low-level Undefined-Phase Cloud Fraction',  &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+    
+!       ! Calipso Opaque/thin cloud diagnostics
```

Comment:
Any particular reason why these are commented out?

Reply:
Copied from the original COSP interface. Apparently whoever implemented this originally didn't want to enable these diagnostics, but probably copied them over when going through everything available? I'm not sure.

Reply:
Removed commented out sections here.